addons/spreadsheet/static/src/list/plugins/list_core_plugin.js

```diff
@@ -70,6 +70,11 @@ export default class ListCorePlugin extends CorePlugin {
                     return CommandResult.EmptyName;
                 }
                 break;
+            case "UPDATE_ODOO_LIST_DOMAIN":
```

Comment:
you can put this under `RENAME_ODOO_LIST`

Reply:
Nope, RENAME_ODOO_LIST also checks for `name`

Reply:
yup, i'm blind